discojs/src/client/client.ts

```diff
@@ -40,7 +43,10 @@ export abstract class Client extends EventEmitter<{'status': RoundStatus}>{
    * we were doing before waiting (training locally or updating our model).
    * We use this attribute to store the status to rollback to when we stop waiting
    */
-  private previousStatus: RoundStatus | undefined;
+  #previousStatus: RoundStatus | undefined;
+
+  // Current number of participants in the training session
```

Comment:
I'm never sure if that includes the current client or not, consider adding a small comment to help my brain :)